src/store/cartSlice.ts

```diff
@@ -0,0 +1,103 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+interface CartSliceState {
+  items: {
+    id: number;
+    name: string;
+    price: number;
+    totalPrice: number;
```

Comment:
shouldn't this be a property of all items and not of single ones?

Reply:
we talked about it at the meeting. This is the total price of one item in the cart